nrf_modem/doc/ug_nrf_modem_porting_os.rst

```diff
@@ -127,11 +129,6 @@ This function is called by the Modem library when the library wants to clear IRQ
 
 * Clear the low priority scheduling IRQ of the Modem library using OS primitives or NVIC functions.
 
-nrf_modem_application_irq_handler()
```

Comment:
This does not really fit here, as it is a function to be called, not to be implemented. Information put further down and linked in `irq_set`